lib/resources/r4/related_person.yaml

```diff
@@ -61,9 +61,19 @@ fields:
     - display: Millennium Person Relationship Types
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/351
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-351-person-relationship-types
-    - display: Millennium Person Relationships
-      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/40
-      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-40-person-relationships
+
+- name: relationship relation extension
+  type: CodeableConcept
+  description: The nature of the relationship between a patient and the related person.
+  url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.relationship
```

Comment:
I don't think we need a url since a url doesn't exist for the relation extension.

Reply:
removed because of https://github.com/cerner/fhir.cerner.com/pull/624#issuecomment-826635553